### PR TITLE
TNO-1717: Add isPress and Aliases to the Contributor 

### PR DESCRIPTION
**Pull request by jdtoombs**
Draft until tno-core PRs merged, then this one will be updated.

In this PR:

- added `isPress` flag and `aliases` to the contributor 
- `isPress` and `aliases` editable through the contributor form
- `isPress` displayed on contributor list view
- added contributors to subscriber lookup
- started basic page for PressGallery (start of TNO-1728)

image

image

### CODE REVIEW

**Comment by Fosol** [comment]
You'll need to redo your DB migration once I merge my PR.  Remember how to properly rollback and remove your snapshot?  If not, ask me.

**Comment by jdtoombs** [comment]
@Fosol - updated!

File: app/subscriber/src/features/press-gallery/PressGallery.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { FlexboxTable, IContentModel, ITableInternalRow, Row } from 'tno-core';
+
+import * as styled from './styled';
+
+/** TODO: Didn't realize I had two different stories - this page will be done in the next ticket*/
+export const PressGallery: React.FC = () => {
+  const navigate = useNavigate();
+  // const [items, setItems] = React.useState<IContentModel[]>([]);
```
Comment (Fosol) [inline app/subscriber/src/features/press-gallery/PressGallery.tsx:13]: Any reason to leave these commented out line?

File: libs/net/dal/Migrations/20230921212403_1.0.69.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    public partial class _1069 : SeedMigration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AddColumn<string>(
+                name: "aliases",
+                table: "contributor",
+                type: "text",
```
Comment (Fosol) [inline libs/net/dal/Migrations/20230921212403_1.0.69.cs:18]: You should update the `ContributorConfiguration` to include your new properties and also should limit aliases to 500 max length.

**Review by Fosol — CHANGES_REQUESTED** [review]

File: app/subscriber/src/features/press-gallery/PressGallery.tsx (hunk 1 above)
Comment (jdtoombs) [reply app/subscriber/src/features/press-gallery/PressGallery.tsx:13]: Was just for my next ticket (1728)!

File: libs/net/dal/Configuration/ContributorConfiguration.cs [hunk 3]
```diff
@@ -10,6 +10,8 @@ public override void Configure(EntityTypeBuilder<Contributor> builder)
     {
         builder.Property(m => m.Id).IsRequired().ValueGeneratedOnAdd();
         builder.Property(m => m.SourceId).IsRequired(false);
+        builder.Property(m => m.Aliases).IsRequired(false);
```
Comment (Fosol) [inline libs/net/dal/Configuration/ContributorConfiguration.cs:13]: You need to add the MaxLength here.

**Review by Fosol — CHANGES_REQUESTED** [review]

File: libs/net/dal/Configuration/ContributorConfiguration.cs (hunk 3 above)
Comment (jdtoombs) [reply libs/net/dal/Configuration/ContributorConfiguration.cs:13]: d'oh

File: libs/net/dal/Migrations/20230921212403_1.0.69.cs [hunk 4]
```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using TNO.DAL;
+
+#nullable disable
+
+namespace TNO.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class _1069 : SeedMigration
```
Comment (Fosol) [inline libs/net/dal/Migrations/20230921212403_1.0.69.cs:9]: You did not correctly update your migration to include your latest configuration changes.  Make sure you rollback correctly.  Ask if you forget.

**Review by Fosol — CHANGES_REQUESTED** [review]

File: libs/net/dal/Migrations/20230921212403_1.0.69.cs (hunk 4 above)
Comment (jdtoombs) [reply libs/net/dal/Migrations/20230921212403_1.0.69.cs]: 20th time the charm, updated again

**Review by Fosol — APPROVED** [review]

**Pull request merged**